app/views/layouts/application.html.erb

```diff
@@ -13,6 +13,8 @@
 
   <body>
     <main class="container mx-auto mt-28 px-5 flex flex-col justify-start items-center">
+      <p class="notice"><%= notice %></p>
+      <p class="alert"><%= alert %></p>
```

Comment:
- [ ] When I run the application I got the following error as shown with the screenshot related with **The asset "tailwind.css" is not present in the asset pipeline.** on these line `<%= stylesheet_link_tag "tailwind", "inter-font", "data-turbo-track": "reload" %>` So to run the app properly update the code line with `<%= stylesheet_link_tag "application.tailwind", "inter-font", "data-turbo-track": "reload" %>`.
- The error showed up when I run the app
![image](https://github.com/martinkarugaba/blog_app/assets/93598189/f945b67b-42b8-4f70-98fe-7005cd012d26)